tests/test_parsing.py

```diff
@@ -0,0 +1,34 @@
+from chai_lab.data.parsing.input_validation import (
+    constituents_of_modified_fasta,
+    identify_potential_entity_types,
+)
+from chai_lab.data.parsing.structure.entity_type import EntityType
+
+from .example_inputs import example_dna, example_ligands, example_proteins, example_rna
+
+
+def test_simple_fasta():
+    parts = constituents_of_modified_fasta("RKDES")
+    assert parts is not None
+    assert all(x == y for x, y in zip(parts, ["R", "K", "D", "E", "S"]))
+
+
+def test_modified_fasta():
```

Comment:
thx for adding, can you also add example for RNA and DNA?

Reply:
Done!